scripts/rss/write-xml.js

```diff
@@ -0,0 +1,11 @@
+import fs from 'fs-extra'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'fs-extra'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->